src/pages/be-square/static/style/_what-is-besquare.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
+import { Container, Flex, SectionContainer } from 'components/containers'
+import { Text } from 'components/elements'
+import device from 'themes/device'
+
+export const Section = styled(SectionContainer)`
+    width: 100%;
+    background-color: var(--color-grey-30);
+    padding: 80px 0;
+
+    @media ${device.tablet} {
+        padding: 40px 16px;
+    }
+`
+
+export const ContentContainer = styled(Container)`
+    align-items: center;
+    justify-content: center;
+
+    @media ${device.tabletL} {
+        display: grid;
+        grid-row-gap: 30px;
+    }
+`
+
+export const ContentWrapper = styled(Flex)`
+    flex-direction: column;
+    width: fit-content;
+    margin-right: 24px;
```

Comment:
I suppose this needs to be removed once the layout switches to tablet/mobile to keep the cards properly centered.